custom-targets/util/clouddeploy/clouddeploy.go

```diff
@@ -119,11 +125,6 @@ const (
 	RenderNotSupported RenderStatus = "NOT_SUPPORTED"
 )
 
-// Cloud Deploy known result metadata keys.
-const (
```

Comment:
Why are you removing this? We should be added another constant for `custom-target-source-commit-sha`.